src/buffer_logger/can_buffer_logger.py

```diff
@@ -0,0 +1,81 @@
+import can
+import asyncio
+from buffer_logger.buffer_logger import BufferLogger
+from numpy import array2string, empty, set_printoptions
+from datetime import datetime
+
+
+class CanBufferLogger(BufferLogger):
+    ''' This class is being used to log all 'raw' can messages asychronously.
+    '''
+
+    def __init__(self, config, config_type):
+        super().__init__(config, config_type)
+        can_logging = self.config[self.config_type]['raw_can_data_logging']
+        if(can_logging['enabled']):
+            # Creates new network connection because CANopen library has
```

Comment:
is dit ook nog steeds het geval met async?

Reply:
ahja deze comment is helemaal niet meer van toepassing